service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -801,6 +844,30 @@ protected void doStop(final Promise<Void> stopPromise) {
                 .mapEmpty();
     }
 
+    /**
+     * Starts a service client.
+     * <p>
+     * This method invokes the given client's {@link Lifecycle#start()} method.
+     *
+     * @param serviceClient The client to start.
+     * @param serviceName The name of the service that the client is for (used for logging).
+     * @return A future indicating the outcome of starting the client.
+     * @throws NullPointerException if any of the parameters are {@code null}.
+     */
+    protected final Future<Void> startServiceClient(final Lifecycle serviceClient, final String serviceName) {
+
+        Objects.requireNonNull(serviceClient);
+        Objects.requireNonNull(serviceName);
+
+        return serviceClient.start().map(c -> {
+            log.info(" {} successfully connected to {}", serviceClient, serviceName);
+            return c;
+        }).recover(t -> {
+            log.warn("{} failed to connect to {}", serviceClient, serviceName, t);
```

Comment:
(Leading whitespace in first log)
How about changing the `serviceName` parameter values to "Event"/"Telemetry" and using
````
            log.info("{} client '{}' successfully connected", serviceName, serviceClient);
            return c;
        }).recover(t -> {
            log.warn("{} client '{}' failed to connect", serviceName, serviceClient, t);
````
? Then the `startInternal` method already doesn't mention the AMQP messaging network anymore.